docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -445,18 +408,65 @@ fe80::1%lo0 localhost
-
-### Configuring Traffic to External Resources
-You can improve your overall test performance by disabling these third-party resource calls. If you're using Sauce Connect Proxy, the additional network hops required to access external resources have the potential to slow test execution dramatically. To retrieve resources directly, you can use the [`--direct-domains`](/dev/cli/sauce-connect-proxy/#--direct-domains) flag. To blocklist traffic so it is immediately dropped, use the [`--fast-fail-regexps`](/dev/cli/sauce-connect-proxy/#--fast-fail-regexps) command.
+## Improving Sauce Connect Proxy Performance
+During testing, your website or app may load resources (e.g. tracking services, images/videos, advertisements), which can impact page load times and even cause tests to fail.
+If these resources are not needed at all for testing purposes, you can disable the traffic to improve performance.
 
 See [How to Remove Third Party Resources](http://elementalselenium.com/tips/66-blacklist) for more information.
 
-### Be Aware of How Sauce Connect Proxy Caches Traffic
-By default, Sauce Connect Proxy will cache all traffic with SSL Bumping (see [SSL Certificate Bumping](/secure-connections/sauce-connect/security-authentication)). Caching of resources takes place on the Sauce Labs side, resulting in faster test execution.
+### Tuning Sauce Connect Proxy Traffic
+If you're using Sauce Connect Proxy, the additional network hops required to access external resources have the potential to slow test execution dramatically.
+When Sauce Connect Proxy is used, all the traffic is forwarded over the Sauce Connect Proxy connection.
+The following flags provide fine control over the Sauce Connect Proxy tunneled traffic:
+  * [`--tunnel-domains`](/dev/cli/sauce-connect-proxy/#--direct-domains)
+  * [`--direct-domains`](/dev/cli/sauce-connect-proxy/#--tunnel-domains)
+  * [`--fast-fail-regexps`](/dev/cli/sauce-connect-proxy/#--fast-fail-regexps)
+
+#### Tunnel domains
+[`--tunnel-domains`](/dev/cli/sauce-connect-proxy/#--direct-domains) flag allows to specify requests which should always be forwarded from the Sauce Labs hosted browser to customer-side over the Sauce Connect Proxy connection.
+Starting Sauce Connect Proxy with [`--tunnel-domains`](/dev/cli/sauce-connect-proxy/#--direct-domains) implies that requests that don't match "tunnel domains" will be forwarded over the public internet.
+This is the recommended option for the best performance since it allows to minimize the expensive tunnelled traffic and use it only for the internal domains that are not publicly available.
```

Comment:
`since it allows to minimize the expensive tunnelled traffic`

Slightly awkward sentence... maybe write "since it minimizes the expensive tunnelled traffic and uses it"?